virttest/utils_kernel_module.py

```diff
@@ -132,22 +177,21 @@ def restore(self):
         +-------------------+-+-+-+-+
         """
 
-        if self._loaded_config is not None:
-            cmds = []
-            # TODO: Handle cases were module cannot be removed
```

Comment:
Better to keep this comment since we still not solve the issue.

Reply:
Restored.